azurelinuxagent/ga/update.py

```diff
@@ -138,8 +139,13 @@ def get_update_handler():
     return UpdateHandler()
 
 
+def get_agent_global_update_signal_file():
+    return os.path.join(conf.get_lib_dir(), AGENT_UPDATE_SIGNAL_FILE)
+
+
 class UpdateHandler(object):
     TELEMETRY_HEARTBEAT_PERIOD = timedelta(minutes=30)
+    CLEAN_UPDATE_SIGNAL_PERIOD = timedelta(minutes=15)
```

Comment:
is the name CLEAN_UPDATE_SIGNAL_PERIOD accurate? i think we are cleaning up in the main loop

Reply:
yup we are. This variable is just to signify the period of time we'd wait before cleaning up the signal file. Does this rename sound better `REMOVE_UPDATE_SIGNAL_PERIOD`?